src/main/archetype/it.tests/src/main/java/it/tests/GetPageIT.java

```diff
@@ -0,0 +1,104 @@
+/*
+ *  Copyright 2015 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package ${package}.it.tests;
+
+import com.adobe.cq.testing.client.CQClient;
+import com.adobe.cq.testing.junit.rules.CQAuthorPublishClassRule;
+import com.adobe.cq.testing.junit.rules.CQRule;
+import com.adobe.cq.testing.junit.rules.Page;
+import org.apache.sling.testing.clients.ClientException;
+import org.apache.sling.testing.clients.SlingHttpResponse;
+import org.junit.*;
+
+
+/**
+ * Test that some paths exist out-of-the-box on the author service. This test
+ * showcases some <a
+ * href="https://github.com/adobe/aem-testing-clients/wiki/Best-practices">best
+ * practices</a> of the <a
+ * href="https://github.com/adobe/aem-testing-clients">AEM Testing Clients</a>.
+ */
+public class GetPageIT {
+
+    // The CQAuthorClassRule represents an author service. The rule will read
+    // the hostname and port of the author service from the system properties
+    // passed to the tests.@ClassRule
+
+    @ClassRule
+    public static CQAuthorPublishClassRule cqBaseClassRule = new CQAuthorPublishClassRule();
+
+    // CQRule decorates your test and adds additional functionality on top of
+    // it, like session stickyness, test filtering and identification of the
+    // test on the remote service.
+
+    @Rule
+    public CQRule cqBaseRule = new CQRule(cqBaseClassRule.authorRule, cqBaseClassRule.publishRule);
+
+    // Page will create a test page with a random name and it will make sure
+    // that the page is removed at the end of every test execution. By using a
+    // random name, your test will not conflict with any other test running on
+    // the same instance. By removing the page at the end of the test execution,
+    // you are not going to leave any clutter on the instance under test.
+
+    @Rule
+    public Page root = new Page(cqBaseClassRule.authorRule);
```

Comment:
This one is not needed for this test class and would create and delete a page for every test that just does a GET

Reply:
Fixed at 202fcd8.